src/runtime/cuda/cuda_hardware_manager.cpp

```diff
@@ -325,7 +325,7 @@ cuda_hardware_context::get_property(device_uint_property prop) const {
     return 8; // TODO
     break;
   case device_uint_property::global_mem_cache_line_size:
-    return 128; //TODO
+    return 128; // TODO
```

Comment:
Maybe remove this change from this PR, or open a separate PR for it?

Reply:
All of the minor nits could go into a PR. Also, could you make a GitHub action that ensures each source file ends with a newline? Xcode automatically adds one to any file you view.